### PR TITLE
Helm chart for demo app - Nginx demo

### PR DESCRIPTION
**Pull request by Denislite**

### CODE REVIEW

File: demo-helmchart/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion : v2
+name       : NginxDemo-HelmChart
+description: Helm Chart for demonstration Container Monitor App work
+type       : application
+version    : 0.1.0
```
Comment (vangithub1992) [inline demo-helmchart/Chart.yaml:5]: I'm advising to use an empty line at the end of file

File: demo-helmchart/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.metadata.deploymentName }}
```
Comment (vangithub1992) [inline demo-helmchart/templates/deployment.yaml:4]: maybe we can add some descriptions to YAML fields for better understanding of the process?

**Review by vangithub1992 — APPROVED** [review]

File: demo-helmchart/Chart.yaml (hunk 1 above)
Comment (Denislite) [reply demo-helmchart/Chart.yaml:5]: okey, will be done

File: demo-helmchart/templates/deployment.yaml (hunk 2 above)
Comment (Denislite) [reply demo-helmchart/templates/deployment.yaml:4]: How to do it better? Directly in a template or in a file with variables(values.yaml)?

Comment (vangithub1992) [reply demo-helmchart/templates/deployment.yaml:4]: Yep, You right. I agree, descriptions in values.yaml would be better.

**Review by vangithub1992 — APPROVED** [review]
Great! Thanks

**Review by Kaborda-Irina — APPROVED** [review]

**Pull request merged**